src/peft/peft_model.py

```diff
@@ -1545,3 +1545,62 @@ def _prefix_tuning_forward(
                 hidden_states=outputs.hidden_states,
                 attentions=outputs.attentions,
             )
+
+
+class PeftModelForFeatureExtraction(PeftModel):
```

Comment:
Would be awesome to include example usage in a code snippet like we currently have for the other `PeftModel`s!